harvester_e2e_tests/apis/test_images.py

```diff
@@ -83,6 +93,18 @@ def test_get(self, api_client, unique_name):
         assert 200 == code, (code, data)
         assert unique_name == data['metadata']['name']
 
+    def test_get_and_delete_long_name(self, api_client, unique_name):
+        # Case 1: get all images
+        code, data = api_client.images.get()
+
+        assert len(data['items']) > 0, (code, data)
+
+        # Case 2: get created image with long name
+        code, data = api_client.images.get(("".join(list("".join([unique_name]*6))[0:43])))
```

Comment:
The function signature of `join` is `iterable`, which means we can use string directly without convert it to **list**.
and the
```python
"".join(list("".join([unique_name]*6))[0:43])
```

would equals to

```python3
(unique_name*6)[:43]
```

Reply:
Thanks for that mention :smile: - I guess I had forgot that join was unnecessary :sweat_smile: